### PR TITLE
add a project about VN1 competition

### PR DESCRIPTION
**Pull request by dmwyd**

### CODE REVIEW

**Comment by liu-jc** [comment]
Hi @dmwyd,

Thanks for the contribution! We're glad to see that finetuing moirai can reach the first place in the comppetition you mentioned. May I double check which team you are in the leaderboard? I suppose it's "The Zero Theorem" by taking look at the leaderboard?  https://www.datasource.ai/en/home/data-science-competitions-for-startups/phase-2-vn1-forecasting-accuracy-challenge/leaderboard  

Another point is that could you help to fix the workflow check (i.e., black and isort). We would like to make sure the code nice and clean.

**Comment by dmwyd** [comment]
Sorry, only after the competition ended did I attempt to fine-tune the model using the competition dataset, so Moirai's results were not included in the official leaderboard. However, in reality, with simple hyperparameter adjustments, the fine-tuned model's predictive performance could outperform the top five entries on the leaderboard.

**Review by liu-jc — APPROVED** [review]
LGTM
